src/main/kotlin/io/github/mojira/arisa/infrastructure/config/Arisa.kt

```diff
@@ -9,6 +9,7 @@ object Arisa : ConfigSpec() {
         val username by required<String>()
         val password by required<String>()
         val dandelionToken by required<String>(description = "Token for dandelion.eu")
+        val discordLogWebhook by required<String>(description = "Webhook to post log in a Discord channel")
```

Comment:
Can't you make this optional instead of required so you don't need to put optional=true while reading the config?

Reply:
Actually the `optional = true` part is in case there's no `arisa.yml` file while testing. But anyways the `readConfig` method won't be executed when testing so we can remove that part safely